requirements.txt

```diff
@@ -0,0 +1,40 @@
+absl-py>=0.7.1
+appdirs>=1.4.3
+asn1crypto>=0.24.0
+astor>=0.8.0
+asttokens>=1.1.13  
+beautifulsoup4>=4.7.1
+blinker>=1.4
+certifi>=2018.8.24
+chardet>=3.0.4
+Click>=7.0
+cookies>=2.2.1
+cryptography>=2.8,<3.4.7
+decorator>=4.3.0
+entrypoints>=0.3
+Flask>=1.0.2
+flask-cors
+gast>=0.2.2
+google-pasta>=0.1.7
+# grpcio>=1.26
+h5py>=2.9.0
+idna>=2.6
+itsdangerous>=0.24
+Jinja2>=2.10.1
+matplotlib==3.3.2
+numpy>=1.16.2
+oauthlib>=2.1.0
+Pillow>=5.4.1
+pyOpenSSL>=19.0.0
+PyYAML>=5.1.2
+requests>=2.25
+requests-oauthlib>=1.0.0
+responses>=0.9.0
+simplegeneric>=0.8.1
+simplejson>=3.16.0
+webencodings>=0.5.1
+Werkzeug>=0.15.3
+concurrent-log-handler>=0.9.19
+watchdog>=0.10
```

Comment:
@bhavikapanara please clean up this requirements.txt list. May libs are not required for the standalone fall detection library.

Reply:
done...
In case if I missed to remove any unnecessary packages from the requirements.txt list please let me know.